region/brinstar/red/Red Brinstar Fireflea Room.json

```diff
@@ -1229,12 +1229,23 @@
         ]}
       ]
     },
+    {
+      "link": [4, 1],
+      "name": "IBJ Into HBJ",
+      "requires": [
+        {"or": [
+          "canLongIBJ",
+          "canJumpIntoIBJ"
+        ]},
+        "canHBJ"
```

Comment:
HBJ is meant to be used from the ground, in the air is treated as a special case.
Trying it out, airborne HBJ feels like a short diagonal bomb jump.
It doesn't feel safe enough to me for expert.